resources/function_help/json/affine_transform

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "affine_transform",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns the geometry after an affine transformation. Calculations are in the Spatial Reference System of this geometry. The operations are performed in a scale, rotation, translation order. If there is a Z or M offset but the coordinate is not present in the geometry, it will be added.",
+  "arguments": [
+    {"arg":"geometry","description":"a geometry"},
+    {"arg":"deltaX","description":"x-axis translation"},
+    {"arg":"deltaY","description":"y-axis translation"},
+    {"arg":"rotationZ","description":"rotation around z-axis in degrees"},
```

Comment:
```suggestion
    {"arg":"rotationZ","description":"rotation around z-axis in degrees counter-clockwise"},
```

Reply:
My math lessons are quite old now so bear with me... Does it mean that the rotation is done around z axis on point O(0,0,0) or do I misunderstand?

Reply:
yes it works like that

Reply:
I'm not sure this is what people would expect. (0 0 0) may be faaaar from the area of study/not in the crs extent.
The centroid of the feature (as used by the rotate function) sounds more desirable IMHO. And it will make the tools coherent/compatible.

Reply:
In that case the should use a rotate function -- an affine transformation has a strict definition, we can't change what that is